jkube-kit/jkube-kit-spring-boot/src/main/java/org/eclipse/jkube/springboot/enricher/SpringBootHealthCheckEnricher.java

```diff
@@ -112,8 +114,11 @@ protected Probe buildProbe(Integer initialDelay, Integer period, Integer timeout
               springBootConfiguration.getManagementContextPath() : "";
         } else {
             scheme = StringUtils.isNotBlank(springBootConfiguration.getServerKeystore()) ? SCHEME_HTTPS : SCHEME_HTTP;
-            prefix = StringUtils.isNotBlank(springBootConfiguration.getServerContextPath()) ?
-              springBootConfiguration.getServerContextPath() : "";
+            String basePath = springBootConfiguration.getServerContextPath();
+            if (hasSpringWebFluxDependency(getContext().getProject())) {
+                basePath = springBootConfiguration.getWebFluxBasePath();
```

Comment:
What if the property is not set?
What if the property is not set and getServletContextPath did return something in the previous step? is this possible?

Reply:
If `spring.webflux.base-path` property is not set `basePath` would be null and application health endpoints would not have any additional prefixes.

> What if the property is not set and getServletContextPath did return something in the previous step? is this possible?

From what I read in [Spring Docs](https://docs.spring.io/spring-boot/appendix/application-properties/index.html), these properties are doing same thing but for different type of applications . My assumption is based on this sentence in documentation:
> Relative to the servlet context path (server.servlet.context-path) or WebFlux base path (spring.webflux.base-path) when the management server is sharing the main server port

- `server.servlet.context-path`   : used for defining base path for traditional Servlet applications
- `spring.webflux.base-path` : used for defining base path for reactive applications